light_node/src/main.rs

```diff
@@ -233,10 +233,19 @@ fn block_on_actors(
             .num_of_peers_for_bootstrap_threshold(),
     );
 
+    let mut actor_system_cfg = riker::load_config();
+    if env.riker_threads > 0 {
+        let thread_count = env.riker_threads.min(num_cpus::get());
+        actor_system_cfg
+            .set("dispatcher.pool_size", thread_count as i64)
```

Comment:
@binier 
could you, please, take a look if it is possible to change this:
```
.name_prefix("pool-thread-#") -> riker-thread-#
```

Reply:
Looked into it and not possible. Everything which could give us ability to change that is private and not exposed in public interface by riker.